monitoring/monitorlib/fetch/__init__.py

```diff
@@ -62,7 +65,7 @@ def describe_request(req: requests.PreparedRequest,
 class ResponseDescription(dict):
   @property
   def status_code(self) -> int:
-    return self['code']
+    return self['code'] if self['code'] is not None else 999
```

Comment:
can you get away with self.get('code', 999) ?

Reply:
Not in this case because the main condition we're expecting here is that there is an entry at `'code'` containing `None`.  But re: your comment below, I changed this to be resilient to missing `'code'` by using `get`.